src/mono/wasm/Wasm.Build.Tests/WasmSIMDTests.cs

```diff
@@ -48,13 +48,41 @@ public void BuildWithSIMD_NoAOT_ShouldNotRelink(BuildArgs buildArgs, RunHost hos
         }
 
         [Theory]
-        // https://github.com/dotnet/runtime/issues/75044 - disabled for V8, and NodeJS
-        //[MemberData(nameof(MainMethodTestData), parameters: new object[] { /*aot*/ true, RunHost.All })]
-        [MemberData(nameof(MainMethodTestData), parameters: new object[] { /*aot*/ true, RunHost.Chrome })]
+        [MemberData(nameof(MainMethodTestData), parameters: new object[] { /*aot*/ false, RunHost.All })]
+        public void BuildWithoutSIMD_AOT(BuildArgs buildArgs, RunHost host, string id)
+        {
+            string projectName = $"nosimd_with_workload_aot";
+            buildArgs = buildArgs with { ProjectName = projectName };
+            buildArgs = ExpandBuildArgs(buildArgs, "<WasmEnableSIMD>false</WasmEnableSIMD>");
+
+            (_, string output) = BuildProject(buildArgs,
+                                    id: id,
+                                    new BuildProjectOptions(
+                                        InitProject: () => File.WriteAllText(Path.Combine(_projectDir!, "Program.cs"), s_simdProgramText),
+                                        Publish: false,
+                                        DotnetWasmFromRuntimePack: true));
+
+            if (!_buildContext.TryGetBuildFor(buildArgs, out _))
+            {
+                // Check if this is not a cached build
+                Assert.Contains("Compiling native assets with excc", output);
+            }
```

Comment:
Ilona fixed `BuildProject` to always return the build output. So, this assert can be unconditional. And also the string has a typo - `excc` -> `emcc`.